src/expressions.rst

```diff
@@ -550,6 +550,10 @@ All :t:`[expression]s` that are not :t:`[place expression]s` are
 A :t:`value expression context` is an :t:`expression context` that is not a
 :t:`place expression context`.
 
+:dp:`fls_8uhfwqurbyqf`
+The evaluation of a :t:`value expression` in a :t:`place expression context` may
+produce a :t:`temporary`.
+
```

Comment:
This is obviously not sufficient, but to specify this properly we'll have to go source code diving